nss-day-cohort-45/nutyshell-running-with-scissors-1
# Ar styling tests

# Description
Took a pass at styling, also added "next event" feature to events.
## Type of change
# Testing Instructions
- serve app and db
- make sure nothing broke and that everything looks cool
- observe that first chronological event appears different than others
# Checklist:
